Mathlib/Algebra/Module/Defs.lean

```diff
@@ -233,12 +241,17 @@ protected theorem Module.nontrivial (R M : Type*) [Semiring R] [Nontrivial M] [A
     [Module R M] : Nontrivial R :=
   MulActionWithZero.nontrivial R M
 
--- see Note [lower instance priority]
-instance (priority := 910) Semiring.toModule [Semiring R] : Module R R where
+instance (priority := 910) [NonUnitalSemiring R] : NonUnitalModule R R where
```

Comment:
```suggestion
instance (priority := 910) NonUnitalSemiring.toNonUnitalModule
    [NonUnitalSemiring R] : NonUnitalModule R R where
```

Reply:
The auto-generated name is `instNonUnitalModule`, which is indeed not ideal :)